src/helpers/tests/geoChart/fixtures.ts

```diff
@@ -75,3 +75,77 @@ export const COLOR_ITEM: VisualizationObject.IBucket = {
         },
     ],
 };
+
+export const LOCATION_ITEM_BY_IDENTIFIER: VisualizationObject.IBucket = {
+    localIdentifier: LOCATION,
+    items: [
+        {
+            visualizationAttribute: {
+                localIdentifier: "a_State",
+                displayForm: {
+                    identifier: "label.location",
+                },
+            },
+        },
+    ],
+};
+
+export const SEGMENT_BY_ITEM_BY_IDENTIFIER: VisualizationObject.IBucket = {
+    localIdentifier: SEGMENT,
+    items: [
+        {
+            visualizationAttribute: {
+                localIdentifier: "a_Type",
+                displayForm: {
+                    identifier: "label.segment",
+                },
+            },
+        },
+    ],
+};
+
+export const TOOLTIP_TEXT_ITEM_BY_IDENTIFIER: VisualizationObject.IBucket = {
+    localIdentifier: TOOLTIP_TEXT,
+    items: [
+        {
+            visualizationAttribute: {
+                localIdentifier: "a_State",
+                displayForm: {
+                    identifier: "label.tooltipText",
+                },
+            },
+        },
+    ],
+};
+
+export const SIZE_ITEM_BY_IDENTIFIER: VisualizationObject.IBucket = {
+    localIdentifier: SIZE,
+    items: [
+        {
+            measure: {
+                localIdentifier: "m_size",
+                definition: {
+                    measureDefinition: {
+                        item: { identifier: "measure.Population" },
+                    },
+                },
+            },
+        },
+    ],
+};
+
+export const COLOR_ITEM_BY_IDENTIFIER: VisualizationObject.IBucket = {
+    localIdentifier: COLOR,
+    items: [
+        {
+            measure: {
+                localIdentifier: "m_color",
```

Comment:
this should be `m_area`, it indicates that Area measure in Color bucket

Reply:
fixed